src/Uuid/Barebones.elm

```diff
@@ -7,15 +7,16 @@ valid Uuid.
 See the main Uuid module docs for more information on how
 
 @docs uuidStringGenerator, isValidUuid
+
 -}
 
-import String
-import List
 import Array
+import Bitwise
 import Char
+import List
+import Random exposing (Generator, Seed, int, list, map, step)
 import Regex
-import Bitwise
-import Random.Pcg exposing (Generator, map, list, int, step, Seed)
+import String
 
 
 {-| Random.PCG Generator for Uuid Strings. Using this Generator instead of the generate
```

Comment:
This comment still references `Random.PCG`. Suggestion for replacement: 

```elm
{-| Random Generator for Uuid Strings. Using this Generator instead of the generate
function lets you create lists of Uuids, map them to other types etc.
-}
```